docs/how-to/refresh/refresh-multi-cluster.md

```diff
@@ -0,0 +1,38 @@
+# How to refresh a multi-cluster deployment
+
+A MySQL multi-cluster deployment (also known as a multi-node cluster or cluster set) can be upgraded by performing a refresh of each cluster individually.
+
+This guide goes over the steps and important considerations before refreshing multiple MySQL clusters.
+
+## Determine cluster order
+
+To upgrade a multi-cluster deployment, each cluster must be refreshed one by one - starting with the standby clusters.
+
+**The primary cluster must be the last one to get refreshed.**
+
+When a primary cluster gets refreshed, it triggers a potentially costly re-election process. To minimise this cost, all standby clusters should be refreshed before the primary.
```

Comment:
While this is true, the bigger value to start on standby is, if there are issues with the upgrade, availability is not affected at all.